tm2/pkg/bft/types/block.go

```diff
@@ -265,8 +258,9 @@ func (h *Header) GetTime() time.Time { return h.Time }
 func MakeBlock(height int64, txs []Tx, lastCommit *Commit) *Block {
 	block := &Block{
 		Header: Header{
-			Height: height,
-			NumTxs: int64(len(txs)),
+			Height:   height,
+			NumTxs:   int64(len(txs)),
+			TotalTxs: int64(len(txs)),
```

Comment:
Just to have this noted, the `TotalTxs` value is the total number of transactions on the chain (including the block being made). This value is set correctly in `Populate`.

I'm cool with leaving it as-is, since some tests call `MakeBlock` and `Populate` separately